### PR TITLE
Update governance.adoc to reflect current terminology

### PR DESCRIPTION
**Pull request by jwiegley**

### CODE REVIEW

File: modules/developers-guide/pages/concepts/governance.adoc [hunk 1]
```diff
@@ -4,28 +4,29 @@
...
 
-As a decentralized platform, all changes to the configuration and behavior of the Internet Computer are controlled by a governance body called the Network Nervous System (NNS). The NNS controls many aspects of the Internet Computer platform including the following:
+As a decentralized blockchain, all changes to the configuration and behavior of the Internet Computer are controlled by a governance body called the Network Nervous System (NNS). The NNS controls many aspects of the Internet Computer blockchain including the following:
 
 - which data center providers participate in the network
 - the number, location, and ownership of the nodes accepted from a data center provider
-- assignment of nodes to subnets
-- whether upgrades to smart contract canister software or management services are allowed or not
+- assignment of nodes to subnet blockchains
+- whether upgrades to canister smart contracts or management services are allowed or not
```
Comment (jancamenisch) [inline modules/developers-guide/pages/concepts/governance.adoc:12]: What are management services? Maybe replace  ..or  to a new protocol version ...

File: modules/developers-guide/pages/concepts/governance.adoc [hunk 2]
```diff
@@ -4,28 +4,29 @@
...
 
 In addition, only members of the governance body can vote to approve or deny requests to upgrade Internet Computer replicas or modify the Internet Computer protocol.
 
 ## Voting rights for stakeholders
 
 Because tokens on the Internet Computer are generally liquid, they do not represent a stable enough commitment on the part of their holders for them to be used for governance purposes. To provide the stability required for responsible governance, tokens can be converted to **neurons**. A neuron represents a number of ICP tokens that cannot be exchanged for a minimum period of time (the lock-up period).
 
-When a person or organization has some number of ICP tokens locked up in a neuron, the neuron holder has the right to vote on governance issues, and to be paid for voting in proportion to the number of ICP locked up and the length of the lock-up period.
+When a person or organization has some number of ICP tokens locked up in a neuron, the neuron holder has the right to vote on governance issues, and to be paid for voting in proportion to the number of ICP locked up, the length of the lock-up period and the percentage of proposal votes participated in each day.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/concepts/governance.adoc:20]: governance issue -> proposals
and the percentage of proposal votes participated in each day -> relative amount of vote cast. (?)

File: modules/developers-guide/pages/concepts/governance.adoc (hunk 1 above)
Comment (jwiegley) [reply modules/developers-guide/pages/concepts/governance.adoc:12]: I prefer that

File: modules/developers-guide/pages/concepts/governance.adoc (hunk 2 above)
Comment (jwiegley) [reply modules/developers-guide/pages/concepts/governance.adoc:20]: I'll say 'relative number of votes cast'.

**Review by krpeacock — APPROVED** [review]

**Pull request merged**